apps/editor/src/js/codeBlockPreview.js

```diff
@@ -2,21 +2,22 @@
  * @fileoverview Implements CodeBlockPreview
  * @author NHN FE Development Lab <dl_javascript@nhn.com>
  */
+import $ from 'jquery';
```

Comment:
Should be this removed if we want to deprecate `jquery`?

Reply:
@elamperti I'm gradually removing JQuery, and this is the code to add temporarily as needed. And this PR is for tracking work, so I don't need to review it.  😅

Reply:
@seonim-ryu my bad :)